pyEpiabm/pyEpiabm/tests/test_unit/test_core/test_person.py

```diff
@@ -61,6 +61,11 @@ def test_update_status(self):
         self.assertEqual(
             self.person.infection_status,
             pe.property.InfectionStatus.InfectMild)
+        self.person.household = MagicMock()
+        self.person.update_status(pe.property.InfectionStatus.Exposed)
+        self.assertEqual(
+            self.person.infection_status,
+            pe.property.InfectionStatus.Exposed)
```

Comment:
I think you are not actually testing the correct thing. In person.py, you call add_susceptible_person/remove_susceptible_person, so here you need to check if len(person.household.susceptible_persons) and also person's household changed or not.